changelog.d/5092.feature

```diff
@@ -0,0 +1,6 @@
+Added possibilty to disable local password authentication
```

Comment:
This file forms part of the [changelog](https://github.com/matrix-org/synapse/blob/master/CHANGES.md) at release, so should be a short summary. Just the first line will do fine.

Feel free to name yourself as a contributor here!